plugin-api/src/main/java/org/hyperledger/besu/plugin/data/BlockHeader.java

```diff
@@ -164,8 +163,16 @@
    *
    * @return TheBASEFEE of this header.
    */
-  @Unstable
```

Comment:
Keep the unstable tag.  Another PR is coming in to change this.

Reply:
done 👍